tests/integration/tier0/single-yum-transaction-validation/test_single_yum_transaction_validation.py

```diff
@@ -0,0 +1,95 @@
+import os
+import shutil
+
+import pytest
+
+from conftest import SYSTEM_RELEASE_ENV
+from envparse import env
+
+
+PKI_ENTITLEMENT_CERTS_PATH = "/etc/pki/entitlement"
+
+
+def backup_entitlement_certs():
+    """
+    Utillity function to backup and remove the entitlment cert as soon as we
+    notice then in the the `PKI_ENTITLEMENT_CERTS_PATH`.
```

Comment:
```suggestion
def remove_entitlement_certs():
    """
    Utility function to remove the entitlement certificate as soon as we
    notice it in the `PKI_ENTITLEMENT_CERTS_PATH`.
    
    We don't need to back it up and then restore it because the PKI_ENTITLEMENT_CERTS_PATH folder is only created during
    the conversion when the subscription-manager package is installed. And the .pem certificate is being generated by
    subscription-manager in the folder during the system registration. So to have the test system clean after the test
    finishes the certs shouldn't be present.
```